src/constraint_system/composer.rs

```diff
@@ -827,12 +827,12 @@ mod tests {
 
     #[allow(unused_variables)]
     #[test]
-    #[ignore]
     /// Tests that an empty circuit proof passes
-    fn test_prove_verify() {
+    fn test_empty_circuit() {
```

Comment:
Its not really empty but with dummy constraints. I believe the name should reflect that